frontend/src/components/ShootMessageDetails.vue

```diff
@@ -47,10 +47,9 @@ limitations under the License.
       <v-list-item-content>
         <v-list-item-subtitle>Last Updated</v-list-item-subtitle>
         <v-list-item-title class="d-flex align-center pt-1">
-          <lazy-component @show="showLastUpdateTimePlaceholder=false">
-              <time-string :dateTime="lastUpdateTime" :pointInTime="-1"></time-string>
-            </lazy-component>
-            <v-progress-circular v-if="showLastUpdateTimePlaceholder" indeterminate size="18" width="1"></v-progress-circular>
+          <v-lazy>
+            <time-string :dateTime="lastUpdateTime" :pointInTime="-1"></time-string>
+          </v-lazy>
```

Comment:
is the long loading times of the time-string no fixed with the new v-lazy component or what was the reason to remove the progress indicator?
```js
<v-progress-circular v-if="showLastUpdateTimePlaceholder" indeterminate size="18" width="1"></v-progress-circular>
```

Reply:
Yes, loading is now lightning fast, don't need a loading indicator anymore :)